oauth-app/src/main/resources/application.yml

```diff
@@ -13,10 +15,62 @@ info:
 server:
   # Authorization HTTP header requires to enlarge max header size.
   max-http-header-size: 65000
+  port: 8082
+
+oauth:
```

Comment:
Wouldn't there be any conflict, that `oauth` key uses twice: in line 20 and in line 60?